pkg/cmd/server/api/types.go

```diff
@@ -54,12 +54,12 @@ var (
 	KubeAPIGroupsToAllowedVersions = map[string][]string{
 		APIGroupKube:           {"v1"},
 		APIGroupExtensions:     {"v1beta1"},
-		APIGroupApps:           {"v1alpha1"},
+		APIGroupApps:           {"v1beta1"},
```

Comment:
are apps/v1alpha1 objects in storage anywhere? is the alpha API version still surfaced?